test/onnx/verify_onnx.cpp

```diff
@@ -631,6 +631,125 @@ TEST_CASE(mean_integral_test)
     EXPECT(migraphx::verify_range(result_vector, gold));
 }
 
+TEST_CASE(mod_test)
+{
+    migraphx::program p = migraphx::parse_onnx("mod_test.onnx");
+    p.compile(migraphx::ref::target{});
+
+    migraphx::shape s{migraphx::shape::float_type, {3, 3, 3}};
+
+    std::vector<float> a = {-4.0, 7.0,  5.0, 4.0,  -7.0, 8.0, -4.0, 7.0,  5.0,
+                            4.0,  -7.0, 8.0, -4.0, 7.0,  5.0, 4.0,  -7.0, 8.0,
+                            -4.0, 7.0,  5.0, 4.0,  -7.0, 8.0, -4.0, 7.0,  5.0};
```

Comment:
Can we use some values with fractions part nonzero ?

Reply:
done